pontoon/base/static/css/heading_info.css

```diff
@@ -49,13 +49,18 @@ ul {
 }
 
 #heading .details .value a {
-  color: #EBEBEB;
+  color: #FFFFFF;
 }
 
 #heading .details .value a:hover {
   color: #7BC876;
 }
 
+#heading .details li:hover .value a {
+  text-decoration: underline;
```

Comment:
I don't like the underline effect here. Chaging color should be enough.

Reply:
@mathjazz done :)